### ISSUE TITLE
Deep Learning for Optimization of Trajectories for Quadrotors. (arXiv:2309.15191v2 [cs.RO] UPDATED)

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#28804]
Deep Learning for Optimization of Trajectories for Quadrotors. (arXiv:2309.15191v2 [cs.RO] UPDATED)<br>
https://ift.tt/NkuWfGU<br>
<p>This paper presents a novel learning-based trajectory planning framework for
quadrotors that combines model-based optimization techniques with deep
learning. Specifically, we formulate the trajectory optimization problem as a
quadratic programming (QP) problem with dynamic and collision-free constraints
using piecewise trajectory segments through safe flight corridors [1]. We train
neural networks to directly learn the time allocation for each segment to
generate optimal smooth and fast trajectories. Furthermore, the constrained
optimization problem is applied as a separate implicit layer for
backpropagation in the network, for which the differential loss function can be
obtained. We introduce an additional penalty function to penalize time
allocations which result in solutions that violate the constraints to
accelerate the training process and increase the success rate of the original
optimization problem. To this end, we enable a flexible number of sequences of
piece-wise trajectories by adding an extra end-of-sentence token during
training. We illustrate the performance of the proposed method via extensive
simulation and experimentation and show that it works in real time in diverse,
cluttered environments.
</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
